### PR TITLE
export: add tooltip for intent.

### PR DESCRIPTION
**Pull request by TurboGit**

### CODE REVIEW

**Comment by TurboGit** [comment]
@aurelienpierre : can you review ? TIA.

File: src/libs/export.c [hunk 1]
```diff
@@ -1303,6 +1303,32 @@ void gui_init(dt_lib_module_t *self)
   dt_bauhaus_combobox_add(d->intent, _("absolute colorimetric"));
   gtk_box_pack_start(GTK_BOX(self->widget), d->intent, FALSE, TRUE, 0);
 
+  tooltip = g_strdup_printf(_("• perceptual : "
+                              "smoothly moves out-of-gamut colors into gamut,"
```
Comment (aurelienpierre) [inline src/libs/export.c:1307]: perceptual is often a proprietary LUT that depends on the destination space. Usually, it compresses the whole gamut range such that the color transitions are preserved indeed.

File: src/libs/export.c [hunk 2]
```diff
@@ -1303,6 +1303,32 @@ void gui_init(dt_lib_module_t *self)
...
+                              "preserving gradations, but distorts in-gamut colors in the process."
+                              "\n\n"
+
+                              "• relative colorimetric : "
+                              "reproduces colors with as little desaturation as possible."
```
Comment (aurelienpierre) [inline src/libs/export.c:1312]: relative colorimetric keeps luminance while reducing saturation until colors fit in gamut

File: src/libs/export.c [hunk 3]
```diff
@@ -1303,6 +1303,32 @@ void gui_init(dt_lib_module_t *self)
...
+                              " space."
+                              "\n\n"
+
+                              "• saturation : "
+                              "designed to present eye-catching business graphics"
+                              " by preserving the saturation. (not suited for photography)."
+                              "\n\n"
+
+                              "• absolute colorimetric : "
+                              "reproduces colors scaled to the white point of the source color"
```
Comment (aurelienpierre) [inline src/libs/export.c:1324]: I would phrase it "adapt white point of the image to the white point of the destination medium and do nothing else"

File: src/libs/export.c (hunk 2 above)
Comment (TurboGit) [reply src/libs/export.c:1312]: Ok, I took this as-is, simpler and less verbose.

File: src/libs/export.c (hunk 3 above)
Comment (TurboGit) [reply src/libs/export.c:1324]: Works for me.

File: src/libs/export.c (hunk 1 above)
Comment (TurboGit) [reply src/libs/export.c:1307]: Ok.

**Comment by TurboGit** [comment]
@aurelienpierre : new version based on your feedback just pushed.

**Review by aurelienpierre — APPROVED** [review]

**Comment by TurboGit** [comment]
Thanks, let's merge.

**Pull request merged**